tests/integ/modin/test_scikit.py

```diff
@@ -0,0 +1,41 @@
+import modin.pandas as pd
+import numpy as np
+import pandas as native_pd
+import pytest
+from numpy.testing import assert_array_equal
+
+import snowflake.snowpark.modin.plugin  # noqa: F401
+from tests.integ.utils.sql_counter import SqlCounter, sql_count_checker
+
+
+# SNOW-1344931 Support MaxAbsScalar by supporting numpy.may_share_memory
+def test_scikit_maxabs():    
+    from sklearn.preprocessing import MaxAbsScaler
```

Comment:
nit: move this import to the top of the file

Reply:
done